schematic/utils/validate_utils.py

```diff
@@ -70,7 +97,22 @@ def rule_in_rule_list(rule: str, rule_list: list[str]) -> Optional[re.Match[str]
     return re.search(rule_type, rule_list_str, flags=re.IGNORECASE)
 
 
-def parse_str_series_to_list(col: pd.Series) -> pd.Series:
+def get_list_robustness(val_rule: str) -> str:
+    """Helper function to extract list robustness from the validation rule.
+    Args:
+        val_rule: str, validation rule string.
+    Returns:
+        list_robutness: str, list robustness extracted from validation rule.
+    """
+    rule_parts = val_rule.lower().split(" ")
+    if len(rule_parts) > 1:
+        list_robustness = rule_parts[1]
+    else:
+        list_robustness = "strict"
+    return list_robustness
+
```

Comment:
I think this function needs more explanation. What is list robustness?

Reply:
I'll add more to these docstrings.

Reply:
I'm going to change this function up a bit to be clearer.